src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -971,8 +974,14 @@ bool AccountStoreSC<MAP>::ParseContractCheckerOutput(
 
       bool hasMap = false;
 
-      if (root["contract_info"].isMember("fields")) {
+      if (root["contract_info"].isMember("fields") ||
+          root["contract_info"].isMember("params")) {
+        // We include contract parameters along with the mutable state.
+        auto stateVars = root["contract_info"]["params"];
         for (const auto& field : root["contract_info"]["fields"]) {
```

Comment:
Because of the `or` check in previous line, wouldn't we need to make sure `fields` is also a member here before we access it?